freeciv-web/pom.xml

```diff
@@ -245,6 +245,21 @@ Rerun the sync-js-hand.js script.
 					<artifactId>maven-antrun-plugin</artifactId>
 					<version>1.8</version>
 					<executions>
+						<execution>
+							<!-- workaround for https://github.com/samaxes/minify-maven-plugin/issues/142 -->
+							<id>source-mapping-fix</id>
+							<phase>initialize</phase>
+							<goals>
+								<goal>run</goal>
+							</goals>
+							<configuration>
+								<target>
+									<delete quiet="true">
+										<fileset dir="${project.build.directory}/freeciv-web/javascript" includes="*.map"/>
```

Comment:
Doesn't remove `webgl/libs/webgl-client.min.js.map`. Either we wipe every map file with `**/*.map` or we specifically declare each one separately.

Reply:
:man_facepalming: